docs/src/design/aux_table/pow2.md

```diff
@@ -131,8 +132,9 @@ $$k_1 \cdot (p' - 256 \cdot p) = 0$$
 
 To aggregate the output into column $z$, we'll enforce the following conditions using a validity constraint for the first row and a transition constraint for the rest of the rows in the cycle.
 
-1. In the first row of each 8-row cycle, the value in $z$ aggregates over the current row.
-2. For all rows in the cycle except the last, the value of $z$ in the next row must equal the aggregation of the output in the next row plus the value of $z$ in the current row.
+1. In the first row of each 8-row cycle, $zp$ should be set to 0. 
+2. For all the rows except the last one, the next value of $zp$ should be the same as the current value of $z$ .
+3. For all rows except the last one, the current output $z$ should equal the output value copied from previous row $zp$ plus the current row $p$ times the aggregation of the decomposed powers. 
```

Comment:
I think this is better - maybe just add a bit to the end for extra clarity: "...times the aggregation of the decomposed power in the current row."

Reply:
Done! Please see [bfa8a00](https://github.com/maticnetwork/miden/pull/232/commits/bfa8a0019bce0ea03c28f1d1f5e83672e00b4066).